Code/GraphMol/ForceFieldHelpers/ANI/ANIForceFieldHelperTest.cpp

```diff
@@ -0,0 +1,70 @@
+//
+//  Copyright (C) 2020 Manan Goel
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include "RDGeneral/test.h"
+#include "catch.hpp"
+
+#include <RDGeneral/Invariant.h>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/FileParsers/MolSupplier.h>
+#include <GraphMol/FileParsers/FileParsers.h>
+
+#include <ForceField/ANI/AtomicContrib.h>
+#include <ForceField/ForceField.h>
+#include <GraphMol/Descriptors/AtomicEnvironmentVector.h>
+#include <Eigen/Dense>
+#include <GraphMol/ForceFieldHelpers/ANI/Builder.h>
+
+using namespace RDKit;
+
+TEST_CASE("Check ANI Force Field builder") {
+  SECTION("ANI-1ccx") {
+    std::string pathName = getenv("RDBASE");
+    std::string filePath = pathName + "/Code/GraphMol/Descriptors/test_data/CH4.mol";
+
+    auto mol = MolFileToMol(filePath, true, false);
+    TEST_ASSERT(mol);
+    int confId = -1;
+    auto field = RDKit::ANI::constructForceField(*mol, "ANI-1ccx", 8);
+    field->initialize();
+    auto numAtoms = mol->getNumAtoms();
+    double *pos;
```

Comment:
It's better to use a managed pointer and initialize it immediately:
```suggestion
    std::unique_ptr<double[]> pos(new double[numAtoms*3]);
```
and then delete line 40.
If you aren't going to do this, you need to be sure to `delete[]` the pointer before it goes out of scope.